message_ix/model/MESSAGE/model_core.gms

```diff
@@ -1659,8 +1673,12 @@ ACTIVITY_CONSTRAINT_LO(node,tec,year,time)$( map_tec_time(node,tec,year,time)
 ***
 ACTIVITY_SOFT_CONSTRAINT_LO(node,tec,year,time)$( soft_activity_lo(node,tec,year,time) )..
     ACT_LO(node,tec,year,time) =L=
-        SUM((vintage,mode)$( map_tec_lifetime(node,tec,vintage,year) AND map_tec_act(node,tec,year,mode,time) ),
-            ACT(node,tec,vintage,year,mode,time) ) ;
+        SUM((vintage,mode,year2)$( map_tec_lifetime(node,tec,vintage,year) AND map_tec_act(node,tec,year,mode,time)
```

Comment:
as above
```suggestion
        SUM((vintage,mode,year2)$( map_tec_lifetime(node,tec,vintage,year2) AND map_tec_act(node,tec,year2,mode,time)
```

Reply:
@danielhuppmann Thanks, this seems to resolve the issue. @volker-krey I have included both changes in the PR i made to your branch.